Gemfile

```diff
@@ -64,7 +64,7 @@ gem "pg",                                              :require => false
 gem "pg-dsn_parser",                  "~>0.1.0",       :require => false
 gem "psych",                          "~>3.1",         :require => false # This can be dropped once we drop ruby 2.5
 gem "query_relation",                 "~>0.1.0",       :require => false
-gem "rails",                          "~>5.2.4", ">=5.2.4.4"
+gem "rails",                          "~>5.2.4", ">=5.2.4.6"
```

Comment:
This will ultimately pull in 5.2.6, right?  Can we just bump the minimum to that?

Reply:
Sure, I wasn't aware of the mimemagic PR:  https://github.com/ManageIQ/manageiq/pull/21215

I can close this if we just want to go with that one

Reply:
Yes, let's go with https://github.com/ManageIQ/manageiq/pull/21215 and backport that to jansa...